next-cloudinary/src/lib/cloudinary.ts

```diff
@@ -32,7 +32,7 @@ let cld;
  * constructCloudinaryUrl
```

Comment:
Line 29 - `let cld` should have the type `:Cloudinary`

But when I tried adding that, I got 2 erros about some of the types below (effect not bein a string etc)

So consider checking it out

Reply:
im wondering if that's because i'm not using those methods as they were designed, i'm passing the transformations directly instead of the imported transformation instances. ill look at this after this PR and see if maybe there's a better method to use